src/pages/Details.js

```diff
@@ -54,9 +54,9 @@ export default function Details() {
             <Col xs={10}>
               <h2>{job.title}</h2>
               <div>
-                {job.tags.map((item) => (
+                {job.tags.map(tag => (
                   <Badge variant="secondary" style={{ marginRight: "10px" }}>
```

Comment:
you added the style but didn't use it here

Reply:
fix this too~